docs/concepts/flatcar-container-linux.md

```diff
@@ -61,3 +61,37 @@ Kubelet mount points on Flatcar Container Linux:
 | /etc/resolv.conf           | /etc/resolv.conf           |           |
 | /opt/cni/bin               | /opt/cni/bin               |           |
 
+### Customization
+
+Flatcar Container Linux can be customized via [Container Linux Configs](https://github.com/coreos/container-linux-config-transpiler/blob/master/doc/configuration.md) (CLC)
+(see some [examples](https://github.com/coreos/container-linux-config-transpiler/blob/master/doc/examples.md)) that are interpreted by [Ignition](https://docs.flatcar-linux.org/ignition/what-is-ignition/).
```

Comment:
I would move the "see some examples" part to the end of the sentence for better readability. I'd avoid breaking "configs... that are interpreted".